ui/src/utils/browser.ts

```diff
@@ -1,4 +1,9 @@
 import { APIError, HTTPStatusError } from "src/adapters/api";
+import { Env } from "src/domain";
+
+export function buildAuthorizationHeader(env: Env) {
+  return `Basic ${window.btoa(`${env.api.username}:${env.api.password}`)}`;
+}
```

Comment:
This does not belong here.

I'd move it to `src/adapters/api/index.ts` since it's a utility related to the API authentication.